rotkehlchen/tests/unit/decoders/test_monerium.py

```diff
@@ -0,0 +1,198 @@
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x99a0618B846D43E29C15ac468Eae06d03C9243C7']])
+def test_burning_monerium_on_eth(database, ethereum_inquirer, ethereum_accounts):
+    tx_hex = deserialize_evm_tx_hash('0x10d953610921f39d9d20722082077e03ec8db8d9c75e4b301d0d552119fd0354')  # noqa: E501
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hex,
+    )
+    timestamp = TimestampMS(1701765059000)
+    amount_str = '1161210.84'
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=171,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH_EURE,
+            balance=Balance(amount=FVal(amount_str)),
+            location_label=ethereum_accounts[0],
+            notes=f'Burn {amount_str} EURe',
+            counterparty=CPT_MONERIUM,
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('polygon_pos_accounts', [['0x762e5f511c219823eeC73C743C8245807A53E123']])
+def test_minting_monerium_on_matic(database, polygon_pos_inquirer, polygon_pos_accounts):
+    tx_hex = deserialize_evm_tx_hash('0xb240acc158fb2cdcdebc7321ca4a96f71b371379e2a78a9a7f27d0718a2e3735')  # noqa: E501
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    with database.user_write() as write_cursor:
+        database.add_asset_identifiers(write_cursor, [A_POLYGON_EURE.identifier])
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=polygon_pos_inquirer,
+        database=database,
+        tx_hash=tx_hex,
+    )
+    timestamp = TimestampMS(1701800519000)
+    amount_str = '95'
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=176,
+            timestamp=timestamp,
+            location=Location.POLYGON_POS,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_POLYGON_EURE,
+            balance=Balance(amount=FVal(amount_str)),
+            location_label=polygon_pos_accounts[0],
+            notes=f'Mint {amount_str} EURe',
+            counterparty=CPT_MONERIUM,
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('polygon_pos_accounts', [['0x0A251dF99A88A20a93876205Fb7f5Faf2E85A481']])
+def test_burning_monerium_on_matic(database, polygon_pos_inquirer, polygon_pos_accounts):
+    tx_hex = deserialize_evm_tx_hash('0xeaed8e3e862a9b41189e9039c825ee57fb80385801b8ac5c3ed70339baf243e5')  # noqa: E501
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    with database.user_write() as write_cursor:
+        database.add_asset_identifiers(write_cursor, [A_POLYGON_EURE.identifier])
```

Comment:
same you don't need this. Remove it in the places where you have it in this file